libs/pages/application/ui/src/lib/components/container/container.tsx

```diff
@@ -18,7 +18,7 @@ import { Application } from 'qovery-typescript-axios'
 import { useLocation, useParams } from 'react-router'
 
 export interface ContainerProps {
-  application: Application
+  application?: Application
```

Comment:
I think it's a not good point to have undefined for this component, we still required to have an Application to render this component

Reply:
Unfortunately, the logic that shows skeleton and spinner are inside the container. If I make the application mandatory, I will not be able to display the container before we have an actual application. And then we don't see any skeleton or loading spinner until the application is fetched.
I'm afraid that I must keep this like this for now, as long as the loading logic is inside the container.